scipy/spatial/transform/rotation.py

```diff
@@ -1990,7 +1990,7 @@ def __call__(self, times):
 
         Parameters
         ----------
-        times : array_like, 1D
+        times : float or array_like
```

Comment:
I think @rgommers just indicated that `array_like` actually includes scalars in one of my recent reviews of a PR from Josh. Is this different because it can only be float? I think I'm getting confused about this, because you recently asked me to do that in my geometric slerp PR and then Ralf indicated differently in another.

Reply:
Definition from https://numpydoc.readthedocs.io/en/latest/format.html#other-points-to-keep-in-mind:
```
array_like : For functions that take arguments which can have not only a type
ndarray, but also types that can be converted to an ndarray (i.e. scalar types, 
sequence types), those arguments can be documented with type array_like.
```

Reply:
I guess documentation rules are not that strict and it's hard to formalize them completely. I wanted to specify that both 1-dimensional arrays and scalars are accepted. I'll keep in mind what Ralf said. I tweaked the description of `times` now.